docs/2-Presets/WordPress.md

```diff
@@ -0,0 +1,234 @@
+# Start a WordPress Project with Docker in 3 Easy Steps
+
+1. Run `kool create wordpress my-project`
+2. Run `kool start`
+3. Update Database Information
+
+> Yes, using **kool** + Docker to create and work on new WordPress projects is that easy!
+
+## Requirements
+
+If you haven't done so already, you first need to [install Docker and the kool CLI](/docs/getting-started/installation).
+
+Also, make sure you're running the latest version of **kool**. Run the following command to compare your local version of **kool** with the latest release, and, if a newer version is available, automatically download and install it.
+
+```bash
+$ kool self-update
+```
+
+> Please note that it helps to have a basic understanding of how Docker and Docker Compose work to use Kool with Docker.
+
+## 1. Run `kool create wordpress my-project`
+
+Use the [`kool create PRESET FOLDER` command](/docs/commands/kool-create) to create your new WordPress project:
+
+```bash
+$ kool create wordpress my-project
+```
+
+Under the hood, this command will create your project directory and automatically run `kool preset wordpress`, which helps you easily set up the initial tech stack for your project using an interactive wizard.
+
+```bash
+$ Preset wordpress is initializing!
+
+? What database service do you want to use [Use arrows to move, type to filter]
+> MySQL 8.0
+  MySQL 5.7
+  PostgreSQL 13.0
+  none
+
+? What cache service do you want to use [Use arrows to move, type to filter]
+> Redis 6.0
+  Memcached 1.6
+  none
+
+? What javascript package manager do you want to use [Use arrows to move, type to filter]
+> npm
+  yarn
+
+? What composer version do you want to use [Use arrows to move, type to filter]
```

Comment:
Indeed we seemed to already have Composer inquiry for creating Wordpress app... does that even make sense? Should we remove?

Reply:
@fabriciojs - I think you're right, that it's not applicable to WP and we should remove. Awaiting agreement from @dbpolito

Reply:
yup, we can remove composer from wordpress...